File: ifElseifElse.cpp
#include<bits/stdc++.h>

using namespace std;

int main() {
    int marks;
    cin>>marks;
    string result;
    if(marks>=90) result = "Grade A";
    else if(marks>=70) result = "Grade B";
    else if(marks>=50) result = "Grade C";
    else if(marks>=35) result = "Grade D";
    else result = "Fail";

    cout<<result<<endl;
    return 0;
}

File: ifElseIfElse.java

import java.util.Scanner;

class ifElseIfElse {
    public static void main(String[] args) {
        Scanner scanner = new Scanner(System.in);
        int marks = scanner.nextInt();

        if(marks>=90) {
            System.out.println("Grade A");
        } else if(marks>=70) {
            System.out.println("Grade B");
        } else if(marks>=50) {
            System.out.println("Grade C");
        } else if(marks>=35) {
            System.out.println("Grade D");
        } else {
            System.out.println("Fail");
        }
    }
}